src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml

```diff
@@ -1,6 +1,6 @@
 = render partial: 'webui/project/breadcrumb_items'
 %li.breadcrumb-item.text-word-break-all
-  = link_to truncate(@package.name, length: 50), package_show_path(@project, @package)
+  = link_to truncate(@package.try(:name) || @package, length: 50), package_show_path(@project, @package)
```

Comment:
What's wrong with always `@package.to_s`?

Reply:
Oh right, I didn't think about this. Good suggestion!